src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -82,6 +82,9 @@ CPU::CPU() {
         if (_cores == 0) {
             _cores = _processors;
         }
+        if (_processors > 0 && _numa_nodes > 0 && _cores > 0) {
+            get_cpu_mapping_from_cores(_processors, _numa_nodes, _cores, _proc_type_table, _cpu_mapping_table);
+        }
```

Comment:
Need logic for `else` for Logical closure, e.g. throw expectation.

Reply:
Ok, done.